src/hct_mis_api/apps/household/admin/individual.py

```diff
@@ -50,6 +51,18 @@
 logger = logging.getLogger(__name__)
 
 
+class IndividualDeliveryMechanismDataInline(admin.TabularInline):
+    model = DeliveryMechanismData
+    extra = 0
+    fields = ("delivery_mechanism_choice", "delivery_mechanism", "data", "is_valid")
+    readonly_fields = ("delivery_mechanism_choice", "delivery_mechanism", "data", "is_valid")
```

Comment:
delivery_mechanism_choice field is legacy and will be dropped, no need to include it here